### PR TITLE
added support for a default_version setting

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

File: DependencyInjection/FOSRestExtension.php [hunk 1]
```diff
@@ -171,6 +171,10 @@ private function loadFormatListener(array $config, XmlFileLoader $loader, Contai
                     'fos_rest.format_listener.media_type.version_regex',
                     $config['format_listener']['media_type']['version_regex']
                 );
+                $container->setParameter(
+                    'fos_rest.format_listener.media_type.default_version',
+                    $config['format_listener']['media_type']['default_version']
+                );
```
Comment (xabbuh) [inline DependencyInjection/FOSRestExtension.php]: I would do something like the following instead:

```php
$container->getDefinition('fos_rest.version_listener')->addArgument($config['format_listener']['media_type']['default_version']);
```

This way you don't have to worry about maintaining container parameters in the future (people might use them).

Comment (lsmith77) [inline DependencyInjection/FOSRestExtension.php]: I guess this applies to tons of other parameters as well ..

Comment (xabbuh) [inline DependencyInjection/FOSRestExtension.php]: Probably, though I think 2.0 would be good release to clean them up (another advantage is that you reduce the size of the compiled container).

Comment (lsmith77) [inline DependencyInjection/FOSRestExtension.php]: sure .. I am open to doing this.

Comment (xabbuh) [inline DependencyInjection/FOSRestExtension.php]: alright, see #1158

**Pull request merged**